components/shared/format/Money.vue

```diff
@@ -41,26 +41,27 @@ export default class Money extends Vue {
   }
 
   private async getFiatValue() {
```

Comment:
remove if unused

Reply:
This could be a nice addition to 'Money' component